### PR TITLE
Vikings image index

### LINKED ISSUE #4: Image Seeds
#### As a developer I want at least 20 images to be created when I setup the database.

__Story__:

Working on this application has been a lot of fun, right? You are fast
becoming a git expert, and an expert at working with a deployment
pipeline. Speaking of development and deployment pipelines, is it not pretty
annoying that everytime a new deployment is created it starts out with an empty
database? It seems like everytime a review application is created we need to
manually add a handful of images in order to manually test the
application. Would it not be great if each deployment was created with a
handful of existing images?

__Acceptance criteria__:
- [ ] After running `rake db:setup` at least 20 unique images are shown in the
  homepage.
- [ ] When a new app is deployed on heroku (e.g., every review app), that app
  automatically has at least 20 unique images on its homepage.

__Dependencies__:
- Image Index

### PR DESCRIPTION
**Pull request by wanyue0714**
Closes #4 

---

The following is a checklist to remind you of the steps to take when opening a
new PR. The horizontal line above and everything below can be removed from the
pull request description.

- [ ] I have replaced `X` above with the associated issue number.
- [ ] I have changed the issue's label to "pull request".
- [ ] I have assigned the designated reviewer to this PR.
- [ ] I have set the appropriate label on this PR.
- [ ] I have written clear commit messages ([ref](http://chris.beams.io/posts/git-commit/)).

If needed, please review the document on the
[Image Sharing Project](https://sites.google.com/a/appfolio.com/eng/new-hire-resources/engineering-academy/ropes/image-sharing-project)
for information on how labels should be used.

### CODE REVIEW

File: test/controllers/images_controller_test.rb [hunk 1]
```diff
@@ -30,4 +25,29 @@ def test_should_get_show_image
...
+        link: 'http://www.sugartours.com/images/fall-foliage-event-anchor-470x320.png' },
+      { title: 'fire_bird',
+        link: 'https://www.skyany.com/data/files/shuaiwei/SW011/1.jpg' },
+      { title: 'plants',
+        link: 'https://i.pinimg.com/originals/28/35/99/28359930ef4ee70928fe417650bbb03d.jpg' }
+    ]
+    Image.create!(images)
+    get images_path
+    assert_response :ok
+    assert_select 'img', 3 do |f|
```
Comment (AgentCyrusJr) [inline test/controllers/images_controller_test.rb]: Minor, we encourage code tells story, I suggest change |f| to something which makes more sense. Because later on, if I look it this code, it is hard at first to understand what 'f' is.

File: test/controllers/images_controller_test.rb [hunk 2]
```diff
@@ -30,4 +25,29 @@ def test_should_get_show_image
...
+      assert f[0].attributes['width'].value.to_i <= 400
```
Comment (AgentCyrusJr) [inline test/controllers/images_controller_test.rb]: 👍

File: app/views/images/show.html.erb [hunk 3]
```diff
@@ -5,7 +5,7 @@
   </p>
   <p>
     Image: <br>
-    <%= image_tag @image.link, size: '100*200' %>
+    <%= image_tag @image.link, size: '400x300' %>
```
Comment (AgentCyrusJr) [inline app/views/images/show.html.erb]: According to AC, 400 on width is the only condition check. Is there any reason to also check the height?
If not, I will suggest use 'width' instead of 'size'

File: app/views/images/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  <h2>List of existing images</h2>
+  <div>
+    <% @images.each do |f| %>
+      <p>
+        title:
+        <%= f.title %>
+      </p>
+      <p>
+        link:
+        <%= image_tag(f.link, size: '200x150') %>
```
Comment (AgentCyrusJr) [inline app/views/images/index.html.erb]: Same here

File: test/controllers/images_controller_test.rb (hunk 1 above)
Comment (wanyue0714) [reply test/controllers/images_controller_test.rb]: Sure, I replaced it with 'image'

File: test/controllers/images_controller_test.rb (hunk 2 above)
Comment (wanyue0714) [reply test/controllers/images_controller_test.rb]: :)

File: app/views/images/show.html.erb (hunk 3 above)
Comment (wanyue0714) [reply app/views/images/show.html.erb]: Sure, I have replaced it with 'width' both on index and show pages

Comment (AgentCyrusJr) [inline app/views/images/show.html.erb]: Another tips for commit, according to this commit, you 'display all images on the index page in desc order', however, you also add changes in show.html which, in my opinion, should be in another commit.
